node/src/service.rs

```diff
@@ -100,6 +100,7 @@ pub fn new_partial(
             justification_import: Some(Box::new(grandpa_block_import.clone())),
             client: client.clone(),
             create_inherent_data_providers: move |_, ()| async move {
+                // XXX how can we pass a <T: Config>??
                 let miner = pallet_cash::internal::miner::InherentDataProvider {};
```

Comment:
in principal you can depend on the `gateway-runtime::Runtime` struct and pass it to the generic here, that binds your node impl to a particular (set of) runtime version(s) though